generated/nidcpower/nidcpower/lcr_load_compensation_spot.py

```diff
@@ -0,0 +1,118 @@
+import ctypes
+import numbers
+import platform
+
+import nidcpower._visatype
+import nidcpower.enums as enums
+
+
+# This class is an internal ctypes implementation detail that corresponds to
+# NILCRLoadCompensationSpot in the C API
+class struct_NILCRLoadCompensationSpot(ctypes.Structure):  # noqa N801
+    if platform.system() == "Windows" and platform.architecture()[0] == "64bit":
+        _pack_ = 8
+    elif platform.system() == "Linux" or platform.system() == "Darwin":
+        _pack_ = 4
+    else:
+        _pack_ = 1
+    _fields_ = [
+        ("frequency", nidcpower._visatype.ViReal64),
+        ("reference_value_type", nidcpower._visatype.ViInt32),
+        ("reference_value_a", nidcpower._visatype.ViReal64),
+        ("reference_value_b", nidcpower._visatype.ViReal64),
+    ]
+
+    def __init__(self, data):
+        super(ctypes.Structure, self).__init__()
+        self.frequency = data.frequency
+        self.reference_value_type = enums.LCRReferenceValueType(data.reference_value_type).value
+        if self.reference_value_type == enums.LCRReferenceValueType.IMPEDANCE.value:
+            self.reference_value_a = data.reference_value.real
+            self.reference_value_b = data.reference_value.imag
+        else:
+            self.reference_value_a = data.reference_value
+            self.reference_value_b = 0.0
+
+
+class LCRLoadCompensationSpot(object):
+    """Constructs a DUT specification for a given frequency to use in LCR load compensation."""
+
+    def __init__(
```

Comment:
As mentioned in https://github.com/ni/nimi-python/pull/1702#discussion_r920916779, such a class constructor design was actually the *result of our previous LCR Python API review meeting*, the rationale mentioned during that meeting was that we want to provide Python users with an easy way to construct an `LCRLoadCompensationSpot` based on the `referenceValueType`, instead of having the users populating the `referenceValueA` and `referenceValueB` by themselves.

Reply:
The people with the design context of the Python APIs (Shreyas, Mark, myself) were not involved in that meeting nor understand the design goals and tradeoffs of nimi-python.

I don't think this is worth it. `__init__` should take in `frequency`, `reference_value_type` and `reference_value`.

Reply:
I have updated the constructor design to the latest suggested version in my latest commit.